xml/storage_multipath.xml

```diff
@@ -63,77 +63,7 @@
-    <member>Intel Multi-Flex</member>
-    <member>LSI/ENGENIO INF-01-00</member>
-    <member>NEC DISK ARRAY</member>
-    <member>NETAPP LUN</member>
-    <member>NEXENTA COMSTAR</member>
-    <member>Pillar Axiom</member>
-    <member>PIVOT3 RAIGE VOLUME</member>
-    <member>SGI IS</member>
-    <member>SGI TP9100, TP 9300</member>
-    <member>SGI TP9400, TP9500</member>
-    <member>STK FLEXLINE 380</member>
-    <member>STK OPENstorage D280</member>
-    <member>SUN CSM200_R</member>
-    <member>SUN LCSM100_[IEFS]</member>
-    <member>SUN STK6580, STK6780</member>
-    <member>SUN StorEdge 3510, T4</member>
-    <member>SUN SUN_6180</member>
+   <simplelist><member>3PARdata VV</member><member>AIX NVDISK</member><member>AIX VDASD</member><member>APPLE Xserve RAID</member><member>COMPELNT Compellent Vol</member><member>COMPAQ/HP HSV101, HSV111, HSV200, HSV210, HSV300, HSV400, HSV 450</member><member>COMPAQ/HP MSA, HSV</member><member>COMPAQ/HP MSA VOLUME</member><member>DataCore SANmelody</member><member>DDN SAN DataDirector</member><member>DEC HSG80</member><member>DELL MD3000</member><member>DELL MD3000i</member><member>DELL MD32xx</member><member>DELL MD32xxi</member><member>DGC</member><member>EMC Clariion</member><member>EMC Invista</member><member>EMC SYMMETRIX</member><member>EUROLOGC FC2502</member><member>FSC CentricStor</member><member>FUJITSU ETERNUS_DX, DXL, DX400, DX8000</member><member>HITACHI DF</member><member>HITACHI/HP OPEN</member><member>HP A6189A</member><member>HP HSVX700</member><member>HP LOGICAL VOLUME</member><member>HP MSA2012fc, MSA 2212fc, MSA2012i</member><member>HP MSA2012sa, MSA2312 fc/i/sa, MCA2324 fc/i/sa, MSA2000s VOLUME</member><member>HP P2000 G3 FC|P2000G3 FC/iSCSI|P2000 G3 SAS|P2000 G3 iSCSI</member><member>IBM 1722-600</member><member>IBM 1724</member><member>IBM 1726</member><member>IBM 1742</member><member>IBM 1745, 1746</member><member>IBM 1750500</member><member>IBM 1814</member><member>IBM 1815</member><member>IBM 1818</member><member>IBM 1820N00</member><member>IBM 2105800</member><member>IBM 2105F20</member><member>IBM 2107900</member><member>IBM 2145</member><member>IBM 2810XIV</member><member>IBM 3303 NVDISK</member><member>IBM 3526</member><member>IBM 3542</member><member>IBM IPR</member><member>IBM Nseries</member><member>IBM ProFibre 4000R</member><member>IBM S/390 DASD ECKD</member><member>IBM S/390 DASD FBA</member><member>Intel Multi-Flex</member><member>LSI/ENGENIO INF-01-00</member><member>NEC DISK ARRAY</member><member>NETAPP LUN</member><member>NEXENTA COMSTAR</member><member>Pillar Axiom</member><member>PIVOT3 RAIGE VOLUME</member><member>SGI IS</member><member>SGI TP9100, TP 9300</member><member>SGI TP9400, TP9500</member><member>STK FLEXLINE 380</member><member>STK OPENstorage D280</member><member>SUN CSM200_R</member><member>SUN LCSM100_[IEFS]</member><member>SUN STK6580, STK6780</member><member>SUN StorEdge 3510, T4</member><member>SUN SUN_6180</member>
```

Comment:
Is there a reason for this to be a blob instead of nicely-formatted with line breaks between each set of 'member' tags?

Reply:
OK, reformatted. The groupings make no sense to me; some companies' products are all on a single line, whereas others, notably IBM's, get 1 line per product.

Do you think I should re-group these into a smaller number of lines, or indeed, into a minimal number of lines, or is it best left alone?

Reply:
That looks way better \o/ The groupings are inconsistent, but IMO as long as the file is reasonably human-readable it's good enough. We have enough to do without going nuts over small details.